### PR TITLE
Add clientId existing secret option

### PR DESCRIPTION
**Pull request by thomaspetit**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description

Currently the OIDC block allows you to use your own kubernetes secret for the `clientSecret`. Sadly no implementation was given for the clientId.  This PR introduces a similar key in the `values` file to what we've seen for the `clientSecret`. As I was hesitant to make a breaking change I kept this one to a minimum.

## Motivation and Context

In both enterprise setups (as well as in my homelab) both `clientId` and `clientSecret`are generated in an automated way by my OIDC provider. The setup that is implemented allows for easy rotation from the UI.

Alternatively we allow for `.Values.env` to be duplicated into an `.Values.envFrom`.

## How to test this PR?

Self explanatory

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by thomaspetit** [comment]
@klauspost @harshavardhana if you mind reviewing this? :grin:

**Comment by arnouthoebreckx** [comment]
Would be a really nice change as our automation can take automatically retrieve updates to the clientId that does not require editing the values file

**Comment by thomaspetit** [comment]
@harshavardhana can you and a colleague of yours approve this or provide comments so I can improve the solution? 😁

File: helm/minio/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Multi-Cloud Object Storage
 name: minio
-version: 5.0.14
+version: 5.1.0
```
Comment (harshavardhana) [inline helm/minio/Chart.yaml]: Please do not change helm chart versions, this is a maintainers job.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: helm/minio/Chart.yaml (hunk 1 above)
Comment (thomaspetit) [reply helm/minio/Chart.yaml]: Really sorry for the delay (my 2 kids held me hostage last week). I'll fix it right away 😁

Comment (thomaspetit) [reply helm/minio/Chart.yaml]: @harshavardhana fixed!

**Comment by thomaspetit** [comment]
@harshavardhana fixed! ☺️

**Review by thomaspetit — COMMENTED** [review]
Can anyone review and approve? 😁

**Comment by arnouthoebreckx** [comment]
@harshavardhana can anyone of your team please have a look? This change can help our setup greatly

**Comment by thomaspetit** [comment]
@shtripat @bh4t can you have a look please?

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**